Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/RequirementNodeData.xml

```diff
@@ -6980,4 +6980,84 @@
+        <OperandArray index="1" value="0"/>
+    </CRequirementEq>
+    <CRequirementGTE id="GTECountUnitAP_BanelingLaunchAmmoCompleteOnlyAtUnit1">
+        <OperandArray index="0" value="CountUnitAP_BanelingLaunchAmmoCompleteOnlyAtUnit"/>
+        <OperandArray index="1" value="1"/>
+    </CRequirementGTE>
+    <CRequirementGTE id="GTECountUnitAP_BanelingLaunchAmmoCompleteOnlyAtUnit3">
+        <OperandArray index="0" value="CountUnitAP_BanelingLaunchAmmoCompleteOnlyAtUnit"/>
+        <OperandArray index="1" value="3"/>
+    </CRequirementGTE>
+    <CRequirementGTE id="GTECountUpgradeAP_AberrationBanelingLaunchCompleteOnly1">
+        <OperandArray index="0" value="CountUpgradeAP_AberrationBanelingLaunchCompleteOnly"/>
+        <OperandArray index="1" value="1"/>
+    </CRequirementGTE>
+    <CRequirementGTE id="GTECountUpgradeAP_AberrationBanelingLaunchLevel2CompleteOnly1">
+        <OperandArray index="0" value="CountUpgradeAP_AberrationBanelingLaunchLevel2CompleteOnly"/>
+        <OperandArray index="1" value="1"/>
+    </CRequirementGTE>
+    <CRequirementLT id="LTCountUnitAP_BanelingLaunchAmmoQueuedOrBetterAtUnit1">
+        <OperandArray index="0" value="CountUnitAP_BanelingLaunchAmmoQueuedOrBetterAtUnit"/>
+        <OperandArray index="1" value="1"/>
+    </CRequirementLT>
+    <CRequirementLT id="LTCountUnitAP_BanelingLaunchAmmoQueuedOrBetterAtUnit3">
+        <OperandArray index="0" value="CountUnitAP_BanelingLaunchAmmoQueuedOrBetterAtUnit"/>
+        <OperandArray index="1" value="3"/>
+    </CRequirementLT>
+    <CRequirementLT id="LTCountUnitAP_ScarabQueuedOrBetterAtUnit5">
+        <OperandArray index="0" value="CountUnitAP_ScarabQueuedOrBetterAtUnit"/>
+        <OperandArray index="1" value="5"/>
+    </CRequirementLT>
+    <CRequirementLT id="LTCountUpgradeAP_AberrationBanelingLaunchLevel2CompleteOnly1">
+        <OperandArray index="0" value="CountUpgradeAP_AberrationBanelingLaunchLevel2CompleteOnly"/>
+        <OperandArray index="1" value="1"/>
+    </CRequirementLT>
+    <CRequirementOr id="OrAndGTECountUpgradeAP_AberrationBanelingLaunchCompleteOnly1LTCountUnitAP_BanelingLaunchAmmoQueuedOrBetterAtUnit1GTECountUpgradeAP_AberrationBanelingLaunchLevel2CompleteOnly1">
+        <OperandArray value="AndGTECountUpgradeAP_AberrationBanelingLaunchCompleteOnly1LTCountUnitAP_BanelingLaunchAmmoQueuedOrBetterAtUnit1"/>
+        <OperandArray value="GTECountUpgradeAP_AberrationBanelingLaunchLevel2CompleteOnly1"/>
+    </CRequirementOr>
+    <CRequirementOr id="OrAndLTCountUpgradeAP_AberrationBanelingLaunchLevel2CompleteOnly1GTECountUnitAP_BanelingLaunchAmmoCompleteOnlyAtUnit1GTECountUpgradeAP_AberrationBanelingLaunchLevel2CompleteOnly1">
+        <OperandArray value="AndLTCountUpgradeAP_AberrationBanelingLaunchLevel2CompleteOnly1GTECountUnitAP_BanelingLaunchAmmoCompleteOnlyAtUnit1"/>
+        <OperandArray value="GTECountUpgradeAP_AberrationBanelingLaunchLevel2CompleteOnly1"/>
+    </CRequirementOr>

```

Comment:
All the IDs should start with `AP_` prefix, for example `AP_LTCountUnitBanelingLaunchAmmoQueuedOrBetterAtUnit3`